templates/partials/drc-support-text.html.twig

```diff
@@ -0,0 +1,4 @@
+<p class="text-muted">
+    For technical questions regarding HealthPro or account management, please contact
+    <a href="mailto: drcsupport@vumc.org">drcsupport@vumc.org</a>
```

Comment:
I know this was in the previous text prior to this PR, but there shouldn't be a space after `mailto:`

```suggestion
    <a href="mailto:drcsupport@vumc.org">drcsupport@vumc.org</a>
```